services/iota_wrappers.go

```diff
@@ -299,48 +309,51 @@ func GetProcessingFrequency() float64 {
 }
 
 // Finds Transactions with a list of addresses. Result in a map from Address to a list of Transactions
-func findTransactions(addresses []giota.Address) (map[giota.Address][]giota.Transaction, error) {
+func findTransactions(addresses []trinary.Hash) (map[trinary.Hash][]transaction.Transaction, error) {
```

Comment:
Out of curiosity, is there a `Trytes` or `Address` data type?

Reply:
There is not more Address data type. And it is replaced by Hash
Hash is Trytes.